### PR TITLE
fixes freddyfazbear bitrunning

### LINKED ISSUE #85197: Freddy Fazbear Jumpscares CyberPolice
## Issue Summary

Title.

When you roll cybercop in the FNAF domain, the bears attack you, unlike other domains with hostile mobs, meaning you become free loot for the runner.

## Reproduction:

Go to Freddy Fazbear's Pizzeria.

Become Cybercop.

Get jumpscared by all the bears, die before 6am and get stuffed into a suit.

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
closes https://github.com/tgstation/tgstation/issues/85197
## Changelog
:cl:
fix: fixed bears in the fazbear pizzeria virtual domain attacking cyber police
/:cl:

### CODE REVIEW

**Comment by projectkepler-ru** [comment]
Eto Freddy Fazbear

File: code/modules/mob/living/basic/space_fauna/bear/_bear.dm [hunk 1]
```diff
@@ -70,6 +70,10 @@
 	honey_food.forceMove(get_turf(src))
 
 //SPACE BEARS! SQUEEEEEEEE~     OW! FUCK! IT BIT MY HAND OFF!!
+/mob/living/basic/bear/bitrunning
+	faction = list(ROLE_SYNDICATE)
+	ai_controller = /datum/ai_controller/basic_controller/trooper
```
Comment (Jacquerel) [inline code/modules/mob/living/basic/space_fauna/bear/_bear.dm:75]: Why the change of AI controller? It's not explained in the PR body.

Comment (Sadboysuss) [reply code/modules/mob/living/basic/space_fauna/bear/_bear.dm:75]: is only the faction flag required? I'll test that.

Comment (jlsnow301) [reply code/modules/mob/living/basic/space_fauna/bear/_bear.dm:75]: Why the syndicate domain? Did hostile not work?

Comment (Sadboysuss) [reply code/modules/mob/living/basic/space_fauna/bear/_bear.dm:75]: > Why the syndicate domain? Did hostile not work?

they were attacking `GLITCH` antagonists so I just copied over the behavior from other domains

Comment (jlsnow301) [reply code/modules/mob/living/basic/space_fauna/bear/_bear.dm:75]: glitch antags (assuming you mean cybercops etc?) should have the npc friendly component

Comment (Sadboysuss) [reply code/modules/mob/living/basic/space_fauna/bear/_bear.dm:75]: > glitch antags (assuming you mean cybercops etc?) should have the npc friendly component

I tested it, before my changes `GLITCH` antags (yes, cybercops, cybertac and net guardian prime) were being attacked by the bears

**Comment by Sadboysuss** [comment]
imma focus on other stuff rn
